docs/data/material/migration/upgrade-to-v7/upgrade-to-v7.md

```diff
@@ -23,3 +23,44 @@ The steps you need to take to migrate from Material UI v6 to v7 are described
 This list is a work in progress.
 Expect updates as new breaking changes are introduced.
 :::
+
+### Lab components moved to the main package
+
+The following `@mui/lab` components and hook have been moved to the main package:
+
+- Alert
+- AlertTitle
+- Autocomplete
+- AvatarGroup
+- Pagination
+- PaginationItem
+- Rating
+- Skeleton
+- SpeedDial
+- SpeedDialAction
+- SpeedDialIcon
+- ToggleButton
+- ToggleButtonGroup
+- usePagination
+
+To keep using these components and hook, import them from `@mui/material` instead of `@mui/lab`.
+
+```diff
+-import Alert from '@mui/lab/Alert';
++import Alert from '@mui/material/Alert';
+
+-import { Alert } from '@mui/lab';
++import { Alert } from '@mui/material';
+```
+
+Use this codemod to automatically update the imports:
+
+<!-- #npm-tag-reference -->
+
+```bash
+npx @mui/codemod@next v7.0.0/lab-removed-components <path/to/folder>
+```
+
+:::warning
+The codemod doesn't cover type imports associated with the components.
+:::
```

Comment:
I don't think it's worth our time to add them, there are a lot of them, and I think users are rarely using them. We can add them in the future if enough people ask.

Reply:
(The default component type, i.e., the `Alert` type for the `Alert` component, is covered as it's "the same" import as the component)